src/app/Http/Controllers/CountryController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+
+namespace App\Http\Controllers;
+
+
+use App\Services\CoronaStatusService;
+use Illuminate\Http\Request;
+use Illuminate\View\View;
+
+class CountryController
+{
+    private CoronaStatusService $coronaStatusService;
+
+    public function __construct(CoronaStatusService $coronaStatusService)
+    {
+        $this->coronaStatusService = $coronaStatusService;
+    }
+
+    public function show(Request $request): View
+    {
+        $request->validate([
+            'country' => ['required', 'min:2','max:2', 'string'],
+        ]);
+
+        $country = mb_strtolower($request->get('country'));
+
+        $responseText = $this->coronaStatusService->isCountryAffected($country) ? 'Affected' : 'Not affected';
+
+        return view('country.index', [
+            'title' => 'Affected countries',
+            'response' => $responseText,
+        ]);
```

Comment:
```suggestion
        return view('country.index')
            ->with('title', 'Affected countries')
            ->with('isAffected', $isAffected);
```
Radau, kad galimas toks būdas grąžinti duomenis. Toks būdas yra skaitomesnis ir patogesnis.

Gali pasiskaityti čia: https://stackoverflow.com/questions/18341792/how-to-pass-data-to-view-in-laravel